packages/os/os.spec

```diff
@@ -48,6 +48,7 @@ Source116: load-kernel-modules.service.in
 Source117: cfsignal.service
```

Comment:
Nit in the commit message: there is an extra `implement`

Reply:
Fixed.